hadoop-hdfs-project/hadoop-hdfs-rbf/src/test/java/org/apache/hadoop/fs/contract/router/web/TestRouterWebHDFSContractRootDirectory.java

```diff
@@ -71,4 +71,9 @@ public void testRmRootRecursive() {
   public void testRmEmptyRootDirRecursive() {
     // It doesn't apply because we still have the mount points here
   }
+
+  @Override
+  public void testSimpleRootListing() {
+    // It doesn't apply because DFSRouter dosn't support LISTSTATUS_BATCH.
```

Comment:
prefer an `Assume.assumeTrue(unsupported, false)` so it goes into the skipped stats. But, given the rest of the tests are the same, not much of an issue.